src/createAdminAccountScript.ts

```diff
@@ -1,88 +1,114 @@
-// Step by step
-// 1. Connect DB
-console.log('Admin entered: ' + process.argv[2]);
-
 // Import functions
 
 import generator from 'generate-password';
-
-
 import nodemailer from 'nodemailer';
+
 import User from './models/User';
 import connectDatabase from './database';
 import { hashPassword } from './helpers/passwordHash';
```

Comment:
I think we can change the file name to just password instead of passwordHash. So inside of helper/password, there is a hashPassword function.
Later on, we might have additional password-related function, we can put it in the same file. So it can look like
`import { hashPassword, isPasswordSecured, somePasswordHelper } from './helpers/password'`